modules/nf-core/happy/sompy/tests/main.nf.test

```diff
@@ -0,0 +1,127 @@
+nextflow_process {
+
+    name "Test Process HAPPY_SOMPY"
+    script "../main.nf"
+    process "HAPPY_SOMPY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "happy"
+    tag "happy/sompy"
+
+    test("homo_sapiens - vcf") {
+        config './nextflow.config'
+        when {
+            process {
+                """
+                input[0] = [
+                        [ id:'test' ], // meta map
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/gatk/paired_mutect2_calls/test_test2_paired_mutect2_calls.vcf.gz', checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/gatk/haplotypecaller_calls/test2_haplotc.vcf.gz', checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/multi_intervals.bed', checkIfExists: true),
+                        []
+                        ]
+                input[1] = [
+                        [ id:'test2' ], // meta map
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta', checkIfExists: true)
+                        ]
+                input[2] = [
+                        [ id:'test3' ], // meta map
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai', checkIfExists: true)
+                        ]
+                input[3] = [[],[]]
+                input[4] = [[],[]]
+                input[5] = [[],[]]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.stats).match("stats_1") },
+                { assert snapshot(process.out.versions).match("versions_1") },
+                { assert snapshot(file(process.out.metrics[0][1]).name).match("metrics_1")},
+                { assert snapshot(file(process.out.features[0][1]).name).match("features_1")}
```

Comment:
Can you put all the snapshots into one: (indentation probably needs fixing)
```suggestion
                { assert snapshot(
                  process.out.stats,
                  process.out.versions,
                  file(process.out.metrics[0][1]).name,
                  file(process.out.features[0][1]).name
                ).match() },
```

Reply:
when I don't separate "versions", lint gives an error. that is why I put it like this.

Reply:
Hmm, shouldn't be an issue like this, I've done it on a number of modules

Reply:
I dont know what was that, but it works out inside one match now